pkg/search/postgres/select.go

```diff
@@ -16,8 +17,27 @@ import (
 	"github.com/stackrox/rox/pkg/postgres/walker"
 	"github.com/stackrox/rox/pkg/search/postgres/aggregatefunc"
 	pgsearch "github.com/stackrox/rox/pkg/search/postgres/query"
+	"github.com/stackrox/rox/pkg/utils"
 )
 
+var scanAPI = newScanAPI(newDBScanAPI(dbscan.WithAllowUnknownColumns(true)))
```

Comment:
So this solution uses `scany`.  Last I checked `scany` is still built upon `pgx4` and there is a desire to remove `scany` and simply go to using `pgx5` which added the ability to scan by struct.  It would be ideal if we could figure out how to do this as a `pgx5` solution and eliminate the use of `scany` now.

Reply:
That's fine.  IMO, the follow could be to simply update our scany dependency so we no longer carry a pgx4 dependency if we haven't already.